data/transition-sites/phe_datag.yml

```diff
@@ -0,0 +1,7 @@
+---
+site: phe_datag
+whitehall_slug: public-health-england
+homepage: https://www.gov.uk/government/organisations/public-health-england
+tna_timestamp: 20150505172621
```

Comment:
We're expecting this timestamp to change in September, I take it, when they've got the new crawl?